test/modules/home/home_view_test.dart

```diff
@@ -238,5 +244,54 @@ void main() {
         expect(delegate.showDetailButtonDidTap, emits(surveys.first));
       });
     });
+
+    describe("it's showSideMenu() is called", () {
```

Comment:
- sometimes you have the () sometimes you don't, let's be consistent.
- `its` not `it's` here I believe